tests/strings/offsets_general.phpt

```diff
@@ -12,21 +12,10 @@ var_dump(isset($string[0][0]));
 var_dump($string["foo"]);
 var_dump(isset($string["foo"]["bar"]));
 var_dump($string{0});
-var_dump($string{1});
-var_dump(isset($string{0}));
-var_dump(isset($string{0}{0}));
-var_dump($string{"foo"});
-var_dump(isset($string{"foo"}{"bar"}));
```

Comment:
In the interest of keeping at least some code coverage I'd suggest leaving this test alone and adding the appropriate number of deprecation warnings.